pkg/minikube/constants/constants.go

```diff
@@ -68,7 +68,7 @@ var DefaultISOSHAURL = DefaultISOURL + SHASuffix
 var DefaultKubernetesVersion = "v1.17.0"
 
 // NewestKubernetesVersion is the newest Kubernetes version to test against
-var NewestKubernetesVersion = "v1.17.0"
+var NewestKubernetesVersion = "v1.18.0-alpha.1"
```

Comment:
Do you mind removing the part of your PR that updates Newest to v1.18? I feel like the PR is doing too much. Otherwise, I think it's good to go.